frontend/awx/common/util/getDocsBaseUrl.ts

```diff
@@ -1,10 +1,106 @@
 import { Config } from '../../interfaces/Config';
 
-export function getDocsBaseUrl(config: Config | null | undefined) {
-  let version = 'latest';
+export interface DocPathDictionary {
+  credentialTypes: string;
+  credentials: string;
+  organizations: string;
+  teams: string;
+  users: string;
+  activityStream: string;
+  applications: string;
+  executionEnvironments: string;
+  managementJobs: string;
+  notifiers: string;
+  topology: string;
+  workflows: string;
+  eeMigration: string;
+  jobTemplateSurveys: string;
+  index: string;
+  hosts: string;
+  inventories: string;
+  constructedInventories: string;
+  managePlaybooksSC: string;
+  projects: string;
+  templates: string;
+  workflowVisualizer: string;
+  workflowVisBuild: string;
+  jobs: string;
+  schedules: string;
+}
+
+export function getDocsBaseUrl(
+  config: Config | null | undefined,
+  doc: keyof DocPathDictionary
+): string {
+  const version = config?.platformVersion || '2.5';
```

Comment:
I'm a bit nervous about hard coded 2.5 here.  Can we make it `latest` or do something so that we don't have the update this with each release?

Reply:
unfortunately the new downstream docs don't support a `latest` path like our old docs did, so this is the safest fallback